x-pack/qa/rolling-upgrade/build.gradle

```diff
@@ -119,6 +119,8 @@ for (Version version : bwcVersions.wireCompatible) {
         setting 'xpack.security.enabled', 'true'
         setting 'xpack.security.transport.ssl.enabled', 'true'
         setting 'xpack.security.authc.token.enabled', 'true'
+        setting 'xpack.security.authc.token.timeout', '60m'
```

Comment:
Why 60 min here?

Reply:
`60m` It is the maximum `timeout`.
I have increased the lifetime of the tokens because the amount of API calls increased, and I don't see a reason not to.